configtelemetry/type.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package configtelemetry
+
+import (
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"go/types"
+	"strings"
+)
+
+var ignoreComments = []string{
+	"telemetry: none",
+	"This field is deprecated and must not be used.",
```

Comment:
Not needed for now, but a more idiomatic comment is "// Deprecated: do not use". We can do that in a future PR or even a HW ticket.

Reply:
Sure, but I can do it since we are here. Not a big deal :)

Reply:
Not a big deal of work, we can do it since we are here 👍